docs/src/cookbooks/cookbook.md

```diff
@@ -404,6 +406,39 @@ class ModuleWithOptionalIO(flag: Boolean) extends Module {
 }
 ```
 
+### How do I create a single input/output wire for a bundle?
+
+If you have a bundle with many elements in a module interface, the interface can become
+quite verbose. To simplify the module interface, the entire bundle
+can be represented with a single wire.
+
+Consider:
+
+```scala mdoc:silent:reset
+import chisel3._
+
+class DooHickeyBundle extends Bundle {
```

Comment:
this is a bit nitpicky, but `DooHickey` is like `Foo` which we use most places, is there a reason to not use that here? I think `DooHickey` might be confusing for people who aren't native (american?) english speakers, while Foo is practically a term of art at this point...